packages/cli/src/lib/combined-environment-variables.ts

```diff
@@ -0,0 +1,93 @@
+import {fileExists} from '@shopify/cli-kit/node/fs';
+import {resolvePath} from '@shopify/cli-kit/node/path';
+import {
+  outputContent,
+  outputInfo,
+  outputToken,
+} from '@shopify/cli-kit/node/output';
+import {readAndParseDotEnv} from '@shopify/cli-kit/node/dot-env';
+
+import {colors} from './colors.js';
+import {pullRemoteEnvironmentVariables} from './pull-environment-variables.js';
+import {getConfig} from './shopify-config.js';
+
+interface Arguments {
+  root: string;
+  shop?: string;
+}
+export async function combinedEnvironmentVariables({root, shop}: Arguments) {
+  const remoteEnvironmentVariables = await pullRemoteEnvironmentVariables({
+    root,
+    flagShop: shop,
+    silent: true,
+  });
+
+  const formattedRemoteVariables = remoteEnvironmentVariables?.reduce(
+    (a, v) => ({...a, [v.key]: v.value}),
+    {},
+  );
+
+  const dotEnvPath = resolvePath(root, '.env');
+  const localEnvironmentVariables = (await fileExists(dotEnvPath))
+    ? (await readAndParseDotEnv(dotEnvPath)).variables
+    : {};
+
+  const remoteKeys = new Set(
+    remoteEnvironmentVariables.map((variable) => variable.key),
+  );
+
+  const localKeys = new Set(Object.keys(localEnvironmentVariables));
+
+  if ([...remoteKeys, ...localKeys].length) {
+    outputInfo(
+      `${colors.bold('Injecting environment variables into mini-oxygen...')}`,
```

Comment:
Minor thing but can we write it as `MiniOxygen` here? Just to be consistent with the other message.
<img width="382" alt="image" src="https://github.com/Shopify/hydrogen/assets/1634092/f4a13ef9-5497-4268-a7b6-16fa4995fe1a">